### DISCUSSION TITLE
https://scratch.mit.edu/developers

### DISCUSSION THREAD

**Discussion by CloudGate1234** [NovaoTech/solaris#15] [Ideas]
I just wanted to say that there is a wonderful resource to help you make this project: https://scratch.mit.edu/developers. On this page, you’ll find information about open source projects created and maintained by the Scratch Team, as well as our thoughts on best practices for designing learning experiences for children.

**Comment by MikeDev101** [comment]
Maybe we should fork the Scratch WWW code and modify it to work with our needs. Give it a modern makeover and swap in TurboWarp-Esque editing functionality, while being able to host and index projects like Adacraft?

**Reply by CloudGate1234** [reply]
that's what I was thinking

**Reply by icarusinthewings** [reply]
That could work. I've been eyeing using Linode for hosting object storage. It is fairly cheap, with a server being $5 a month for a 256gb bucket of storage. I'm fairly new to the concept of object storage, but I think this is what we could use to store objects.
